oracledb-ballerina/tests/02_execute-basic-test.bal

```diff
@@ -0,0 +1,404 @@
+    groups:["execute","execute-basic"],
+    dependsOn: [testInsertTable]
+}
+isolated function testUpdateTable() {
+    Client oracledbClient = checkpanic new(user, password, host, port, database);
+    sql:ExecutionResult result = checkpanic oracledbClient->execute("UPDATE TestExecuteTable SET field2 = 'Hello, ballerina' WHERE field1 = 1");
+    checkpanic oracledbClient.close();
+    test:assertExactEquals(result.affectedRowCount, 1, "Affected row count is different.");
+    test:assertExactEquals(result.lastInsertId, (), "Last Insert Id should be null.");
+}
+
+@test:Config {
+    groups: ["execute", "execute-basic"],
+    dependsOn: [testInsertTable]
+}
+isolated function testInsertTableWithoutGeneratedKeys() {
+    Client oracledbClient = checkpanic new (user, password, host, port, database);
+    sql:ExecutionResult result = checkpanic oracledbClient->execute("Insert into TestCharacterTable (id, col_varchar2)"
+        + " values (20, 'test')");
+    checkpanic oracledbClient.close();
+    test:assertExactEquals(result.affectedRowCount, 1, "Affected row count is different.");
+    var insertId = result.lastInsertId;
+    test:assertTrue(insertId is string, "Last Insert id should be string");
+}
+
+@test:Config {
+    groups: ["execute", "execute-basic"],
+    dependsOn: [testInsertTableWithoutGeneratedKeys]
+}
+isolated function testInsertTableWithGeneratedKeys() {
+    Client oracledbClient = checkpanic new (user, password, host, port, database);
+    sql:ExecutionResult result = checkpanic oracledbClient->execute("insert into TestNumericTable (col_number) values (21)");
+    checkpanic oracledbClient.close();
+    test:assertExactEquals(result.affectedRowCount, 1, "Affected row count is different.");
+    var insertId = result.lastInsertId;
+    test:assertTrue(insertId is string, "Last Insert id should be string.");
+}
+
+type NumericRecord record {|
+    int id;
+    decimal col_number;
+    decimal col_float;
+    decimal col_binary_float;
+    decimal col_binary_double;
+|};
+
+@test:Config {
+   groups: ["execute", "execute-basic"],
+   dependsOn: [testInsertTableWithGeneratedKeys]
+}
+isolated function testInsertAndSelectTableWithGeneratedKeys() {
+   Client oracledbClient = checkpanic new (user, password, host, port, database);
+   sql:ExecutionResult result = checkpanic oracledbClient->execute("insert into TestNumericTable (col_number) values (31)");
+
+   test:assertExactEquals(result.affectedRowCount, 1, "Affected row count is different.");
+
+   string|int? insertedId = result.lastInsertId;
+   if (insertedId is string|int) {
+        string query = "SELECT * from TestNumericTable where col_number = 31";
+        stream<record{} , error> queryResult = oracledbClient->query(query, NumericRecord);
+
+        stream<NumericRecord, sql:Error> streamData = <stream<NumericRecord, sql:Error>>queryResult;
+        record {|NumericRecord value;|}? data = checkpanic streamData.next();
+
+        checkpanic streamData.close();
```

Comment:
Shall we avoid using `checkpanic`. instead, let's adds `check` and returns error

Reply:
fixed with 57294320259b2614da3d4192b8456f89431641e4